lib/Analysis/SelectVariableNames/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_mlir_dialect_library(MLIRHEIRAnalysisSVN
+        SelectVariableNames.cpp
+
+        ADDITIONAL_HEADER_DIRS
+        ${CMAKE_CURRENT_SOURCE_DIR}
+
+	LINK_LIBS PUBLIC
```

Comment:
Looks like a mix of tabs, spaces, and the space-indented parts don't all have the same space indentation. Can you `expandtab`? Or maybe something like

```bash
sed -i 's/^\s\{2,\}/    /g' **/CMakeLists.txt
```

Reply:
I something similar - fixed up now.